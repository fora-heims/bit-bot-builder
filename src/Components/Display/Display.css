.display {
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 10px;
  margin: 20px;
}

.image-container {
  display: flex;
  flex-direction: column;
  width: 500px;
  height: 500px;
  border: black solid 5px;
  border-radius: 10px;
}
