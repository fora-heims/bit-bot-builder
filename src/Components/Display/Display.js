import './Display.css';
import React from 'react';

export default function Display() {
  return <div></div>;
}
