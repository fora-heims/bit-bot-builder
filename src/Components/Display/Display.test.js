import { render, screen } from '@testing-library/react';
import Display from './Display';

test('renders the Display panel', () => {
  const container = render(
    <Display
      lightCount="5"
      midCount="3"
      darkCount="2"
      feelings=""
      feel=""
      light="orange"
      mid="green"
      dark="blue"
    />
  );
  expect(container).toMatchSnapshot();
});
