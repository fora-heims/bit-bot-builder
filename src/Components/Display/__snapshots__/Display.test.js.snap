// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Display panel 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="display"
      >
        <div
          class="image-container"
        >
          <img
            src="/assets/orange.png"
          />
          <img
            src="/assets/green.png"
          />
          <img
            src="/assets/blue.png"
          />
        </div>
        <span>
          You have changed the light color 
          5
           times.
        </span>
        <span>
          You have changed the middle color 
          3
           times.
        </span>
        <span>
          You have changed the dark color 
          2
           times.
        </span>
        <span>
          words to describe the picture:
        </span>
        <div
          class="feelings"
        >
          <span>
            hello
          </span>
          <span>
            goodbye
          </span>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="display"
    >
      <div
        class="image-container"
      >
        <img
          src="/assets/orange.png"
        />
        <img
          src="/assets/green.png"
        />
        <img
          src="/assets/blue.png"
        />
      </div>
      <span>
        You have changed the light color 
        5
         times.
      </span>
      <span>
        You have changed the middle color 
        3
         times.
      </span>
      <span>
        You have changed the dark color 
        2
         times.
      </span>
      <span>
        words to describe the picture:
      </span>
      <div
        class="feelings"
      >
        <span>
          hello
        </span>
        <span>
          goodbye
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
